workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -78,21 +72,17 @@ private static void Generate()
         {
             try
             {
-                EditorApplication.LockReloadAssemblies();
-
-                // Ensure that all dependencies are in place.
-                if (DownloadCoreSdk.TryDownload() == DownloadResult.Error)
+                if (!Common.CheckDependencies())
                 {
                     return;
                 }
 
-                CopySchema();
+                EditorApplication.LockReloadAssemblies();
 
                 var projectPath = Path.GetFullPath(Path.Combine(Common.GetThisPackagePath(),
                     CsProjectFile));
 
-                var schemaCompilerPath =
-                    Path.GetFullPath(Path.Combine(Application.dataPath, SchemaCompilerRelativePath));
+                var schemaCompilerPath = SchemaCompilerPath;
```

Comment:
Unnecessary I think?

Reply:
The switch below alters `schemaCompilerPath`, so I had to make a copy to avoid altering the original.